<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClassificationSettingsWidget</class>
 <widget class="QWidget" name="ClassificationSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="classificationHorizontalLayout">
     <item>
      <widget class="QLabel" name="classificationLabel">
       <property name="text">
        <string>Classificaiton method:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="classificationComboBox">
       <item>
        <property name="text">
         <string>Low/High frequency</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="classificationStackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="LowHighFrequencyPage">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QLineEdit" name="frequencyThresholdEdit">
         <property name="text">
          <string>30000</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="highLabelNameEdit">
         <property name="text">
          <string>high</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lowLabelNameLabel">
         <property name="text">
          <string>Low frequency label</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="frequencyThresholdLabel">
         <property name="text">
          <string>Frequency threshold [Hz]</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="highLabelNameLabel">
         <property name="text">
          <string>High frequency label</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lowLabelNameEdit">
         <property name="text">
          <string>low</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </item>
   <item>
    <widget class="Line" name="upperHorizontalLine">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
